src/config/lang/english.go

```diff
@@ -77,11 +77,11 @@ const (
 	// zarf connect list
 	CmdConnectListShort = "Lists all available connection shortcuts"
 
-	CmdConnectFlagName       = "Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6"
-	CmdConnectFlagNamespace  = "Specify the namespace.  E.g. namespace=default"
-	CmdConnectFlagType       = "Specify the resource type.  E.g. type=svc or type=pod"
-	CmdConnectFlagLocalPort  = "(Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000"
-	CmdConnectFlagRemotePort = "Specify the remote port of the resource to bind to.  E.g. remote-port=8080"
+	CmdConnectFlagName       = "Specify the resource name.  E.g. name=unicorns or name=unicorn-pod-7448499f4d-b5bk6. Ignored if --name is unset"
```

Comment:
Probably an oversight, but `Ignored if --name is unset` is part of the `--name` flag description but shouldn't be

Reply:
My bad, will fix that